### DISCUSSION TITLE
SSO causes user password error

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#9306] [Archive]
## 01 SSO newly created user

1. Use google openid to verify login and create a new user.
2. Log in to the user and set a password for the user.
3. Log in with the user's email address and password.

## 02 Users that already exist in the database

1. Modify the directus_users data table, and specify the Provider as google for the existing users, and the External Identifier as the user email
2. Log in with the user's email address and password.

The following error will be prompted at this time:

![image](https://user-images.githubusercontent.com/8030134/139579616-6dd43699-84ca-4bd0-9252-c5a82e04c44c.png)

**Comment by aidenfoxx** [comment]
This isn't a bug. If a user has an oauth provider, they cannot log in with the regular auth flow.

**Reply by tan-tan-kanarek** [reply]
Can you explain, where is this oauth provider configured?
Is it in the user account level, organization level or other?

**Reply by aidenfoxx** [reply]
Account level in Directus, under "Provider" and "External Identifier".

**Comment by tan-tan-kanarek** [comment]
Awesome, thank you.
